icpc-wf/ansible/roles/scoreboard/tasks/main.yml

```diff
@@ -0,0 +1,119 @@
+---
+# These tasks configure a static scoreboard
+
+- name: create static scoreboard directory
+  file:
+    path: /home/domjudge/scoreboard
+    state: directory
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+
+- name: create directories for styling, scripts, fonts and flags
+  file:
+    path: /home/domjudge/scoreboard/{{ item }}
+    state: directory
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+  with_items:
+  - css
+  - js
+  - webfonts
+  - flags
+
+- name: download required assets
+  get_url:
+    url: "{{ DOMSERVER_URL }}/{{ item }}"
+    dest: /home/domjudge/scoreboard/{{ item }}
+    owner: domjudge
+    group: domjudge
+    mode: 0644
+  with_items:
+  - favicon.ico
+  - css/bootstrap.min.css
+  - css/fontawesome-all.min.css
+  - js/jquery.min.js
+  - js/bootstrap.bundle.min.js
+  - js/domjudge.js
+  - style_domjudge.css
+  - webfonts/fa-brands-400.ttf
+  - webfonts/fa-brands-400.woff2
+  - webfonts/fa-regular-400.ttf
+  - webfonts/fa-regular-400.woff2
+  - webfonts/fa-solid-900.ttf
+  - webfonts/fa-solid-900.woff2
+  - webfonts/fa-v4compatibility.ttf
+  - webfonts/fa-v4compatibility.woff2
+
+- name: install contest images
+  synchronize:
+    src: files/domjudge-public/
+    dest: "/home/domjudge/scoreboard/"
+    owner: no
+    use_ssh_args: true
+  become: true
```

Comment:
What is the actual difference? When I did this my VSCode linter complained it expects a boolean and not a string.

Reply:
YAML wants `true`, but actually `true` != `True` for the underlying python.

So out of the options the `true` is actually the most erroprone (according to this blog I read yesterday about it)

I think that when you change the filetype to ansible instead of yaml that your issue is gone.